scripts/helpers.js

```diff
@@ -60,6 +60,11 @@ function Module(name) {
   this.name = name;
   this.directory = path.join(ROOT_DIR, 'packages', name);
 
+  if (!test('-e', this.directory)) {
```

Comment:
I think this is the wrong spot, if I had a guess I would say you want to filter out deleted modules [here](https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/scripts/helpers.js#L135).

Reply:
Thanks for that!